nodejs/eks/cert-thumprint.ts

```diff
@@ -67,6 +67,7 @@ async function getThumbprint(issuerUrl: string, retriesLeft: number, interval: n
                 hostname: issuerUrl,
                 port: 443,
                 rejectUnauthorized: false,
+                agent: new https.Agent({ maxCachedSessions: 0 }),
```

Comment:
That’s pretty subtle! :smile: Possibly a comment here on why we need to set this?